client/src/index.js

```diff
@@ -0,0 +1,27 @@
+const { buildResults } = require("./helpers.js");
+const form = document.querySelector("form");
+
+form.addEventListener("submit", onClick);
+
+async function onClick(e) {
+  // console.log(form[0].value);
+  e.preventDefault();
+  localStorage.setItem("search", form[0].value);
+  location.href = "/search.html";
```

Comment:
Remember to add the . in the path "./search.html". It didnt let us load the results on the browser without it.